test/util/login-helpers.js

```diff
@@ -52,16 +51,29 @@ function initUserMock(level) {
 }
 /* eslint-enable camelcase */
 
-function logIn(client, url, level) {
-  initUserMock(level);
+let tokenCounter = 0;
+
+function getUserToken() {
+  return `token-${process.pid}-${tokenCounter++}`;
+}
+
+function logIn(token, client, url, level) {
+  initUserMock(token, level);
+
   client
     .url(`${E2eHelpers.baseUrl}${url}`)
     .waitForElementVisible('body', Timeouts.normal);
```

Comment:
Unfortunately setting the users' session prior to waiting for the body to show on this page provided inconsistent results. I'd love if there were a way to set the session on the first page request so we didn't need to make two for every login. This may be possible by passing a session parameter in the URL or finding a way for the react application to read the users' session id from a method or the redux state instead of directly from local storage.